modules/configuring_hyperthreading_for_a_cluster.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc
+
+[id="configuring_hyperthreading_for_a_cluster_{context}"]
+= Configuring hyperthreading for a cluster
+
+To configure hyperthreading for an {product-title} cluster, set the CPU threads in the performance profile to the same cores that are configured for the reserved or isolated CPU pools.
+
+[NOTE]
+====
+If you configure a performance profile, and subsequently change the hyperthreading configuration for the host, ensure that you update the CPU `isolated` and `reserved` fields in the `PerformanceProfile` yaml to match the new configuration.
+====
+
+[WARNING]
+====
+Disabling a previously-enabled host hyperthreading configuration can cause the CPU core IDs listed in the `PerformanceProfile` yaml to be incorrect. This incorrect configuration can cause the node to become unavailable because the listed CPUs can no longer be found.
+====
+
+.Prerequisites
+
+* Access to the cluster as a user with the cluster-admin role.
+* Install the OpenShift CLI (oc).
+
+.Procedure
+
+. Ascertain which threads are running on what CPUs for the host you want to configure.
++
+You can view which threads are running on the host CPUs by logging in to the cluster and running the following command:
++
+[source,terminal]
+----
+$ lscpu --all --extended
+----
++
+.Example output
++
+[source,terminal]
+----
+CPU NODE SOCKET CORE L1d:L1i:L2:L3 ONLINE MAXMHZ    MINMHZ
+0   0    0      0    0:0:0:0       yes    4800.0000 400.0000
+1   0    0      1    1:1:1:0       yes    4800.0000 400.0000
+2   0    0      2    2:2:2:0       yes    4800.0000 400.0000
+3   0    0      3    3:3:3:0       yes    4800.0000 400.0000
+4   0    0      0    0:0:0:0       yes    4800.0000 400.0000
+5   0    0      1    1:1:1:0       yes    4800.0000 400.0000
+6   0    0      2    2:2:2:0       yes    4800.0000 400.0000
+7   0    0      3    3:3:3:0       yes    4800.0000 400.0000
+----
++
+In this example, there are eight logical CPU cores running on four physical CPU cores. CPU0 and CPU4 are running on physical Core0, CPU1 and CPU5 are running on physical Core 1, and so on.
```

Comment:
CPU1, and CPU5

Reply:
This is incorrect I think Ashley. "CPU0 and CPU4 are running on physical Core0" is a clause, and "CPU1 and CPU5 are running on physical Core 1" is a clause.  Adding a comma after CPU1 breaks up the clause improperly.